terraform/modules/diego/diego_elb_main.tf

```diff
@@ -48,6 +48,8 @@ resource "aws_elb" "diego_elb_main" {
   subnets         = var.elb_subnets
   security_groups = [aws_security_group.diego-elb-sg.id]
 
+  enable_deletion_protection  = true
```

Comment:
I don't think this property is supported on ELB resources?

https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/elb

Reply:
I think It may still work since ultimately its still a load balancer type. Looking more into it.

https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/lb

Reply:
@dtamasang2 Actually, we have a Concourse job that runs to validate our Terraform. And as [you can see here](https://ci.fr.cloud.gov/builds/101844362), that job is failing with errors like:

```
│ Error: Unsupported argument
│ 
│   on ../../modules/elasticache_broker_network/elb.tf line 7, in resource "aws_elb" "elasticache_elb":
│    7:   enable_deletion_protection  = true
│ 
│ An argument named "enable_deletion_protection" is not expected here.
```

There is apparently a difference between what is supported by [`aws_elb` resources](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/elb), like the one in this error, and [`aws_lb` resources](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/lb).